x-pack/plugins/global_search_bar/public/lib/result_to_option.tsx

```diff
@@ -22,7 +22,10 @@ export const resultToOption = (
   const { tagIds = [], categoryLabel = '' } = meta as { tagIds: string[]; categoryLabel: string };
   // only displaying icons for applications and integrations
   const useIcon =
-    type === 'application' || type === 'integration' || type.toLowerCase() === 'enterprise search';
+    type === 'application' ||
+    type === 'integration' ||
+    type.toLowerCase() === 'enterprise search' ||
+    type.toLowerCase() === 'search';
```

Comment:
Note to @sixstringcode this will affect our telemetry data. When we look at click metrics for the Global Search Bar telemetry, the `type` value for Enterprise search data could be `enterprise search` or `search`.